ico/cmd/tokenvault.py

```diff
@@ -135,6 +137,13 @@ def load(chain, web3: Web3, address: str, csv_file: str, token: Contract, addres
 
         print("Row", i, "giving", tokens, "to", addr, "vault", token_vault.address, "time passed", time.time() - start_time, "ETH spent", spent)
 
+        if not is_checksum_address(addr):
+            if skip_checksums:
```

Comment:
can we change this name `skip_checksums`. `skip_checksums` suggests that all addresses that are not checksummed  will be skippped. However, the code forces non-checksummed addresses to checksum.
How about `force_checksum`, `override_checksum`?